### ISSUE TITLE
Feature: Support computed properties for sort-objects and sort-object-keys

### ISSUE THREAD

**Issue by jaketodaro** [azat-io/eslint-plugin-perfectionist#132]
### What rule do you want to change?

sort-objects/sort-object-keys

### Describe the problem

Would it be possible to support sorting computed properties in objects and types?

### Code example

```
enum SomeEnum {
    foo = 'foo',
    bar = 'bar'
}

const myMap: Record<SomeEnum, string> = {
    // sort these keys?
    [SomeEnum.foo]: 'foo thing',
    [SomeEnum.bar]: 'bar thing'
}
```

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
<img width="546" alt="image" src="https://github.com/azat-io/eslint-plugin-perfectionist/assets/5698350/dbc44935-0e82-4132-9772-f7356a60a1b6">

Can you tell me what your problem is? We currently have a rule that sorts objects by key.

Or are you proposing to sort by value? This is not currently implemented and is not planned.

**Comment by jaketodaro** [comment]
My mistake. After an eslint upgrade this works as expected for me. Previously I was getting an error running eslint with`sort-object-types` turned on and I assumed that the error was due to the fact that I was using computed properties for keys, but that doesn't seem to be the case.
